cmd/boost/deal_cmd.go

```diff
@@ -86,6 +86,11 @@ var dealFlags = []cli.Flag{
 		Name:  "wallet",
 		Usage: "wallet address to be used to initiate the deal",
 	},
+	&cli.BoolFlag{
+		Name:  "AnnounceToIPNI",
```

Comment:
Resolving this as we are renaming it to "skip-announce-to-ipni"

Reply:
I think we should leave the CLI command parameter as "announce-to-ipni".
We're changing the name of the field that is sent over the network. But I think for the CLI it's less confusing to have `--announce-to-ipni=true` rather than `--skip-announce-to-ipni=false`.